src/SfxWeb/src/app/views/node/essentials/essentials.component.scss

```diff
@@ -0,0 +1,8 @@
+.placement-constraint-wrapper {
```

Comment:
Since this class is used in two places, we could place it into Style.scss and selectively overwrite or add some attributes in the component level scss files.